cmd/kepler/main.go

```diff
@@ -163,5 +163,14 @@ func createCPUMeter(logger *slog.Logger, cfg *config.Config) (device.CPUPowerMet
 	if fake := cfg.Dev.FakeCpuMeter; fake.Enabled {
 		return device.NewFakeCPUMeter(fake.Zones, device.WithFakeLogger(logger))
 	}
-	return device.NewCPUPowerMeter(cfg.Host.SysFS)
+
+	if len(cfg.Rapl.Zones) > 0 {
+		logger.Info("Filtering RAPL zones", "zones", cfg.Rapl.Zones)
```

Comment:
`cfg.Rapl.Zones` are the zones we want to keep, not the ones we want to filter out.
better say it as enabled zones, like mentioned in comment below
```
 # zones to be enabled, empty enables all default zones
```

Reply:
comment was not just about t he log but about replacing the word `filter` with `enable` or `select`.
instead of `device.WithZoneFilter` we have `device.WithSelectedZones` or `device.WithEnabledZones`
instead of `filterZones` we say `selectZones` or `enabledZones`

but this is a minor comment. we can change it in a follow-up PR if needed.